plugins/main/public/controllers/management/components/management/cdblists/views/list-editor.tsx

```diff
@@ -243,6 +267,37 @@ class WzListEditor extends Component {
     ];
   };
 
+  /**
+   * Validate that if starts with a quote it ends with a quote and vice versa.
+   * Validate that the value has 0 or 2 quotes.
+   * @param {String} value
+   */
+  private validateQuotes(value: string, field: FieldTypes): boolean {
+    const hasMiddleQuotes = value.slice(1, -1).includes('"');
+    const startsWithQuote = value.startsWith('"');
+    const endsWithQuote = value.endsWith('"');
+
+    const isValid =
+      (startsWithQuote && endsWithQuote && !hasMiddleQuotes) ||
+      (!startsWithQuote && !endsWithQuote && !value.includes('"'));
+    if (!isValid) {
+      this.setState({
+        isInvalid: [
+          ...this.state.isInvalid,
+          {
+            field,
+            message: 'Must start and end with quotes or have no quotes at all',
```

Comment:
Define a constant and substitute it accordingly.

Reply:
Done [310743b](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7171/commits/310743b1942f73af2d2e2df3efd88ba0be294136)